conf/modules.config

```
@@ -162,4 +162,26 @@ process {
         ]
     }
 
-}
+    withName: SGDEMUX {
+        publishDir = [
+            [
+                path: { "${params.outdir}/${meta.id}/" },
+                mode: params.publish_dir_mode,
+                pattern: "output/*.{json,csv,html,log,tsv}",
+                saveAs: { filename -> filename.minus("output/") }
+            ],
+            [
+                path: { "${params.outdir}/${meta.id}/" },
+                mode: params.publish_dir_mode,
+                pattern: "output/Samples/*/*.{json,fastq.gz}",
```

Not outputting JSON. And can't the .fastq.gz be merged with the publishDir pattern above?
```suggestion
                pattern: "output/*.{fastq.gz}",
```

---

Resolved in 870389dcf0ca16b276d49b56165f091396425560